### ISSUE TITLE
Add support to build to a different device than the source is located

### ISSUE THREAD

**Issue by tst-rwildes** [google/earthenterprise#958]
It is sometimes needed to have the build output placed on a device other than where the source code is.  This can be for performance reasons where the source code is located on a device with slow iops.

**Comment by tst-ccamp** [comment]
@tst-rwildes Do we still need to add documentation for these options?

**Comment by tst-rwildes** [comment]
Yes.  I'll work on that.  Two new _optional_ parameters where added to allow for more flexible builds.  No current behavior changes if those parameters are not added.
